src/extension.ts

```diff
@@ -105,10 +105,7 @@ export function activate(context: vscode.ExtensionContext) {
     }
 
     const line = range.start.line
-    const lineText =
-      vscode.window.activeTextEditor !== undefined
-        ? vscode.window.activeTextEditor.document.lineAt(line).text.trim()
-        : 'Text Not Found'
+    const lineText = text.trim()
```

Comment:
Now the editlist contains the changes instead of the entire line. So if I write "hello" then the edit list will just container "o" instead of the entire line. Was that what you intended? It looks kind of weird, imo.

Reply:
ah... fixed this localy and never pushed!!